### ISSUE TITLE
Bootloop Realme c3

### ISSUE THREAD

**Issue by GirishKumarSharma** [bkerler/mtkclient#893]
Hi,

After successful all steps of boot unlocking and complete rooting (patching magisk boot image), I just installed an app to uninstall App Market and Game Centre apps including some google stuff i.e. Gmail, Chrome, Google, all got uninstalled too, but now I am in bootloop. Its a Realme c3 RMX 2027 device. I have taken the backup of all the participations, please tell me how do I came up this bootloop issue.

**Comment by GirishKumarSharma** [comment]
Any update please @bkerler ?

**Comment by bkerler** [comment]
I assume the cause was a firmware ota update

**Comment by GirishKumarSharma** [comment]
@bkerler 
I am even not able to boot even after reverting to stock boot.img and vbmeta.img. It is still stucking at Orange stage.
Should I flash all partitions as I think its problem with Magisk with MTK as you mentioned at https://github.com/bkerler/mtkclient/issues/264 too.

**Comment by GirishKumarSharma** [comment]
Huh?
After waiting around 10 minutes at Orange state, it auto boot into recovery, I simply said to reboot, again it is in Orange state after displaying 4 line message. Let us see, now how many times it goes to recovery, now I will wipe the data and reboot the phone and update you here.

**Comment by GirishKumarSharma** [comment]
Finally, I erased all partitions and write back from earlier backup. All partitions successfully written and now device is in normal mode. No root, no unlock boot loader.

**Comment by GirishKumarSharma** [comment]
Finally, I erased all partitions and write back from earlier backup. All partitions successfully written and now device is in normal mode. No root, no unlock boot loader.

**Comment by GirishKumarSharma** [comment]
There is no issue in neither Magisk nor OTA update. After coming back to normal state of phone, I just did following :
1.Unlocked bootloader (1.python mtk e metadata,userdata,md_udc  2. python mtk da seccfg unlock)
(I did nothing in between unlocking bootloader and root) i.e. your next command should be of writing boot.patched
2.Root (python mtk w boot,vbmeta boot.patched,vbmeta.img.empty)
3.python mtk reset (You Done! No issue, no bootloop, nothing else!)
